proto/config/netinst.proto

```diff
@@ -107,4 +114,19 @@ message NetworkInstanceConfig {
 
   // static DNS entry, if we are running DNS/DHCP service
   repeated ZnetStaticDNSEntry dns = 41;
+  // Enable to use DHCP to automatically propagate routes for uplink subnets
+  // into applications connected to them indirectly through local network instances.
+  // This option is only valid for local network instances. For other types
+  // of network instances, it will be ignored.
+  bool propagate_connected_routes = 42;
+  // List of IP routes statically added to the network instance routing table.
+  // Statically routed subnets are also propagated to connected applications
+  // using DHCP, with gateway set to the network instance bridge IP if it is
+  // outside of the network instance subnet.
+  // IP route gateway may point to an external endpoint (provided that network
+  // instance is not air-gapped), or to an IP address of one of the applications
+  // connected to the network instance.
+  // This option is only valid for local network instances. For other types
+  // of network instances, it will be ignored.
+  repeated IPRoute static_routes = 43;
```

Comment:
Will we allow setting witn the propagate flag and providing static routes?
Makes sense to add to the comment whether this is allowed or not.

Reply:
from some of the previous discussion, we don't allow those automatically, user has to define them

Reply:
Sorry, had an imporant typo on my comment.

Reply:
i think they can work independently, good to comment on this api.

Reply:
Done, comment extended.